ostree/ostree_dest.go

```diff
@@ -119,23 +119,33 @@ func (d *ostreeImageDestination) PutBlob(stream io.Reader, inputInfo types.BlobI
 	return types.BlobInfo{Digest: computedDigest, Size: size}, nil
 }
 
-func fixUsermodeFiles(dir string) error {
+func fixFiles(dir string, usermode bool) error {
 	entries, err := ioutil.ReadDir(dir)
 	if err != nil {
 		return err
 	}
 
 	for _, info := range entries {
 		fullpath := filepath.Join(dir, info.Name())
-		if info.IsDir() {
-			if err := os.Chmod(dir, info.Mode()|0700); err != nil {
+		if info.Mode()&(os.ModeNamedPipe|os.ModeSocket|os.ModeDevice) != 0 {
+			// Ignore the chmod error
+			os.Chmod(fullpath, info.Mode()|0600)
+			if err := os.Remove(fullpath); err != nil {
```

Comment:
oh my bad, I assumed it was necessary to rm write protected files in usermode but obviously it is not.  Pushed a new version